tests/sca_package/test_runner.py

```diff
@@ -134,59 +134,39 @@ def test_prepare_and_scan(mocker: MockerFixture, scan_result):
     assert runner._code_repo_path == EXAMPLES_DIR
 
 
-def test_prepare_and_scan_sca_package_scan_disabled(mocker: MockerFixture, scan_result):
```

Comment:
why did you delete it?

Reply:
This test is verifying that the feature flag around sca package scanning is working and returns an empty result.
Since we wanted to delete the feature flag around this feature, I deleted the test, otherwise it would fail.